apps/portal/js/wizard.js

```diff
@@ -146,7 +154,7 @@ $(".pager .finish").click(function() {
             type: $("#dsList option:selected").attr("data-type"),
             filter: $("#txtFilter").val(),
             columns: columns,
-            maxUpdateValue: 10,
+            maxUpdateValue: defaultMaxValue,
```

Comment:
@keizer619 
Is it possible for the user to configure this maxUpdateValue if needed? i.e. can a user specify a value of 100 instead of the default 10? Also, what would happen when the maxUpdateValue is set to 0?